diagnostic.md

```diff
@@ -32,6 +46,18 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+setter
+$('input[name=username]').val('BenBlahBlah');
+the .val is the setter, setting the value of the input
+$('p').text('yahoo');
+sets all p text to yahoo
+
+getter
+$('p')
+the getter method gets all <p>
+$( '.classy' )
+gets all elements with the class of classy
+
```

Comment:
by getter we mean `$('p').text()` to get whatever the text is of a p as an example